app/experimenter/experiments/api/v5/serializers.py

```diff
@@ -319,9 +267,42 @@ def validate_channel(self, value):
         return value
 
 
-class NimbusStatusUpdateSerializer(
-    NimbusChangeLogMixin, NimbusStatusRestrictionMixin, serializers.ModelSerializer
+class NimbusExperimentUpdateSerializer(
+    NimbusChangeLogMixin,
+    NimbusStatusRestrictionMixin,
+    NimbusExperimentOverviewMixin,
+    NimbusExperimentBranchMixin,
+    NimbusExperimentProbeSetMixin,
+    NimbusAudienceUpdateMixin,
+    serializers.ModelSerializer,
 ):
+    name = serializers.CharField(min_length=0, max_length=255, required=False)
+    slug = serializers.ReadOnlyField()
+    application = serializers.ChoiceField(
+        choices=NimbusExperiment.Application.choices, required=False
+    )
+    public_description = serializers.CharField(
+        min_length=0, max_length=1024, required=False, allow_blank=True
+    )
+    hypothesis = serializers.CharField(min_length=0, max_length=1024, required=False)
+    reference_branch = NimbusBranchSerializer(required=False)
+    treatment_branches = NimbusBranchSerializer(many=True, required=False)
+    feature_config = serializers.PrimaryKeyRelatedField(
+        queryset=NimbusFeatureConfig.objects.all(),
+        allow_null=True,
+        required=False,
+    )
+    primary_probe_set_ids = serializers.PrimaryKeyRelatedField(
+        many=True, queryset=NimbusProbeSet.objects.all(), required=False
+    )
+
+    secondary_probe_set_ids = serializers.PrimaryKeyRelatedField(
+        many=True, queryset=NimbusProbeSet.objects.all(), required=False
+    )
+    population_percent = serializers.DecimalField(
+        7, 4, min_value=0.0, max_value=100.0, required=False
+    )
```

Comment:
This *might* be a pre-existing bug that I just noticed:

When filling everything out, using `0.00002` for `population_percent` on the Audience page results in a missing details warning preventing review - but the page isn't named in the sidebar.

I see it properly coming up in the `readyForReview` property of the `getExperiment` GraphQL response.  (i.e. too many decimal places). That what makes me think it's not this PR doing it. That, and I can't find any change here that would cause it.

A fix is probably to add `population_percent` to the `fieldPageMap.audience` list here: 

https://github.com/mozilla/experimenter/blob/main/app/experimenter/nimbus-ui/src/hooks/useExperiment.tsx#L12-L18

Reply:
Another probably unrelated-to-this-PR issue:

Just noticed that I can submit `0.00007` for `population_percent` - client-side seems to report the save succeeded, but there are submit errors reported in the GQL response. I think there's probably a snake_case / camelCase mismatch going on with our new `useCommonForm` hook - i.e. `population_percent` vs `populationPercent`

Reply:
Oh interesting.  Are there issues filed for those?

Reply:
Hmm, now that I dig around, I think arguably these problems fall under issue #4289 and #4291, maybe also adjacent to #4298

Reply:
Oh good yes, so if we finish those off and still find weirdness we can file additional followups.  Thanks for checking @lmorchard !